consensus/enclave/api/src/fee_map.rs

```diff
@@ -112,7 +112,14 @@ impl FeeMap {
             .iter()
             .find(|(_token_id, fee)| (**fee >> SMALLEST_MINIMUM_FEE_LOG2) == 0)
         {
-            return Err(Error::InvalidFee(*token_id, *fee));
+            return Err(Error::InvalidFeeTooSmall(*token_id, *fee));
+        }
+
+        if let Some((token_id, fee)) = minimum_fees
+            .iter()
+            .find(|(_token_id, fee)| (**fee % (1 << SMALLEST_MINIMUM_FEE_LOG2)) != 0)
+        {
+            return Err(Error::InvalidFeeNotDivisible(*token_id, *fee));
```

Comment:
Mainly the same concerns I expressed on the original PR, that requiring integer multiples of 128 given specific tokenomics could be awkward or maybe expensive if token prices vs. fiat go way up. Given that's **purely** hypothetical at the given moment though, this seems a-ok.